posthog/api/user.py

```diff
@@ -455,6 +458,30 @@ def redirect_to_site(request):
     return redirect("{}#__posthog={}".format(app_url, state))
 
 
+@authenticate_secondarily
+def redirect_to_website(request):
+    team = request.user.team
+    app_url = request.GET.get("appUrl") or (team.app_urls and team.app_urls[0])
+
+    if not app_url:
+        return HttpResponse(status=404)
+
+    if not team or not urllib.parse.urlparse(app_url).hostname in ['localhost', 'posthog.com']:
```

Comment:
`['localhost', 'posthog.com']` should be absrtacted out into a constant somewhere so that it's reusable and if it needs updating, only needs to be updated once.

Reply:
done